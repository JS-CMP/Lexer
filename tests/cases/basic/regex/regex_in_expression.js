var str = "Hello World";

// var result = str.match(/[A-Z]/g); NOT HANDLED BY THE LIB

