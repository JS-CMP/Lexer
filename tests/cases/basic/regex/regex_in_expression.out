#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"
#include "customOperators/CustomOperators.hpp"

int main() {
    JS::Any str;
    str = JS::Any(u"Hello World");
    return 0;
}
